categories/git.json

```diff
@@ -1,12 +1,21 @@
 [
-    {
-        "name": "Learn Git Branching",
-        "description": "Game-like website to learn and master git.",
-        "url": "https://learngitbranching.js.org",
-        "free": true,
-        "tags": [
-            "git",
-            "game"
-        ]
-    }
+  {
+    "name": "Learn Git Branching",
+    "description": "Game-like website to learn and master git.",
+    "url": "https://learngitbranching.js.org",
+    "free": true,
+    "tags": [
+      "git",
+      "game"
+    ]
+  },
+  {
+    "name": "Oh Shit, Git!?!",
```

Comment:
Can we censor this to "S***" please (family friendly website)!

Reply:
Done, and updated the link too - hope it handles the encoded URL correctly 😇

Reply:
That’s super cool!